tests/acceptance/course-page/request-language-test.js

```
@@ -66,7 +66,9 @@ module('Acceptance | course-page | request-language-test', function (hooks) {
     signIn(this.owner, this.server);
 
     await catalogPage.visit();
-    await catalogPage.clickOnCourse('Build your own Docker');
+    await this.pauseTest();
```

ESLint

[error] Do not commit `pauseTest()`

(ember/no-pause-test)